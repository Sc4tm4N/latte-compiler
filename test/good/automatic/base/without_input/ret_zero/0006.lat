// print int variables according to int min (compared to C)
int main() {
  int INT_MIN = -2147483648;
  printInt(INT_MIN + INT_MIN);
  printInt(INT_MIN - INT_MIN);
  printInt(INT_MIN + 2 * INT_MIN);
  printInt(INT_MIN - 2 * INT_MIN);
  printInt(7 * INT_MIN);
  printInt(INT_MIN * 7);
  printInt(INT_MIN / 7);
  printInt(7 / INT_MIN);
  printInt(7 % INT_MIN);
  printInt(INT_MIN % 7);
  printInt(INT_MIN * INT_MIN);
  printInt(INT_MIN * INT_MIN * (INT_MIN * INT_MIN) / INT_MIN - INT_MIN * INT_MIN);
  printInt(INT_MIN * INT_MIN * (INT_MIN * INT_MIN) / INT_MIN + INT_MIN * INT_MIN + 5);
  return 0;
}