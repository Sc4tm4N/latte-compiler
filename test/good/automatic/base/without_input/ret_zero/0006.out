0
0
-2147483648
-2147483648
-2147483648
-2147483648
-306783378
0
7
-2
0
0
5
